src/litdata/streaming/dataset.py

```diff
@@ -62,15 +62,24 @@ def __init__(
             seed: Random seed for shuffling.
             serializers: The serializers used to serialize and deserialize the chunks.
             max_cache_size: The maximum cache size used by the StreamingDataset.
+            subsample: Float representing fraction of the dataset to be randomly sampled (e.g., 0.1 => 10% of dataset).
 
         """
         super().__init__()
         if not isinstance(shuffle, bool):
             raise ValueError(f"Shuffle should be a boolean. Found {shuffle}")
 
+        if not isinstance(subsample, float) or subsample < 0 or subsample > 1:
```

Comment:
```suggestion
        if not isinstance(subsample, float) or subsample <= 0 or subsample > 1:
```

If doesn't make sense to request none of it ;)